src/xcode/fastlane/Fastfile

```diff
@@ -207,12 +207,25 @@ platform :ios do
       --form 'file=@../ENA.ipa'"
     )
 
-    # Trigger Tests on Mobile Device Cloud via CircleCI (cwa-app-automation)
-    sh("curl -u '#{ENV["CIRCLE_API_ADMIN_TOKEN"]}': \
-      -d '{\"branch\":\"main\",\"parameters\":{\"mobile-os\":\"ios\"}}' \
-      -H 'content-type: application/json' \
-      https://circleci.com/api/v2/project/github/corona-warn-app/cwa-app-automation/pipeline"
+    # Trigger Tests on Mobile Device Cloud via GitHub Action on Repo cwa-app-automation
+    sh("
+      curl \
+      -X POST \
+      -H 'Accept: application/vnd.github.v3+json' \
+      -H 'authorization: Bearer #{ENV["GITHUB_ACCESS_TOKEN"]}' \
+      -d '{\"ref\":\"main\", \"inputs\": { \"mobile-os\":\"ios\"}}' \
+      https://api.github.com/repos/corona-warn-app/cwa-app-automation/actions/workflows/31092010/dispatches"
+    )
+
+    sh("
+      curl \
+      -X POST \
+      -H 'Accept: application/vnd.github.v3+json' \
+      -H 'authorization: Bearer #{ENV["GITHUB_ACCESS_TOKEN"]}' \
+      -d '{\"ref\":\"main\", \"inputs\": { \"mobile-os\":\"ios125\"}}' \
```

Comment:
What does iOS125 mean here? for version 12.5?

Reply:
is an extra test automation for iOS 12.5 I guess. @thomasaugsten: can you comment on that?

Reply:
Yes correct it's a separate flow to run a complete test flow with iOS 12.5